wcc-contentful-app/spec/requests/wcc/contentful/app/pages_controller_spec.rb

```diff
@@ -26,6 +29,19 @@
     expect(assigns(:page)).to eq(page)
   end
 
+  it 'allows overloading the Page model' do
+    allow(::WCC::Contentful::Model).to receive(:resolve_constant)
+      .with('page')
+      .and_return(MyPage)
```

Comment:
I think this "allow" is just making explicit what's already happening.  You don't have it in the other specs, and it's still calling into MyPage.